onnxruntime/core/framework/partial_graph_execution_state.cc

```diff
@@ -81,15 +83,15 @@ StreamExecutionContext& PartialGraphExecutionState::GetExecutionContext(gsl::spa
         execution_plan->num_barriers,
         device_streams,
         feed_mlvalue_idxs,
-        feeds,
+        std::move(feeds),
```

Comment:
Hope this won't be used by multiple strams.

Reply:
For multiple streams, I think it should not be a problem. Feel free to elaborate more on the potential issues that could happen. :)